docker-for-mac/faqs.md

```diff
@@ -175,11 +107,54 @@ VirtualBox or VMWare Fusion.
 
 ### What is the benefit of HyperKit?
 
-It is thinner than VirtualBox and VMWare fusion, and the version we include is
-tailor made for Docker workloads on the Mac.
+HyperKit is thinner than VirtualBox and VMWare fusion, and the version we include is tailor made for Docker workloads on the Mac.
 
 ### Why is com.docker.vmnetd running after I quit the app?
 
 The privileged helper process `com.docker.vmnetd` is started by `launchd` and
 runs in the background. The process does not consume any resources unless
 Docker.app connects to it, so it's safe to ignore.
+
+## Feedback
+
+### What kind of feedback are we looking for?
+
+Everything is fair game. We'd like your impressions on the download-install
+process, startup, functionality available, the GUI, usefulness of the app,
+command line integration, and so on. Tell us about problems, what you like, or
+functionality you'd like to see added.
+
+### What if I have problems or questions?
+
+You can find information about diagnosing and troubleshooting common issues in the [Logs and Troubleshooting](troubleshoot) topic.
+
+If you do not find a solution in Troubleshooting, browse issues on
+[Docker Desktop for Mac issues on GitHub](https://github.com/docker/for-mac/issues){: target="_blank" class="_"} or create a new one. You can also create new issues based on diagnostics. To learn more, see
+[Diagnose problems, send feedback, and create GitHub issues](troubleshoot.md#diagnose-problems-send-feedback-and-create-github-issues).
+
+The [Docker Desktop for Mac forum](https://forums.docker.com/c/docker-for-mac){: target="_blank" class="_"}
+provides discussion threads as well, and you can create discussion topics there,
+but we recommend using the GitHub issues over the forums for better tracking and
+response.
+
+### How can I opt out of sending my usage data?
+
+If you do not want to send of usage data, use the Stable channel. For more
+information, see [What is the difference between the Stable and Edge versions of Docker Desktop](#stable-and-edge-channels).
+
+### How is personal data handled in Docker Desktop?
+
+When uploading diagnostics to help Docker with investigating issues, the
+uploaded diagnostics bundle may contain personal data such as usernames and IP
+addresses. The diagnostics bundles are only accessible to Docker Inc. employees
```

Comment:
s/Docker Inc./Docker, Inc.

Reply:
Fixed this and all other occurrences in this topic and also in the Windows topic.